### DISCUSSION TITLE
Getting my transcription English script instead of Hindi for a Hindi audio

### DISCUSSION THREAD

**Discussion by Sarathharidas** [openai/whisper#229] [Q&A] [answered]
Hey Team - 

When I try to transcribe Hindi audio, I get the transcription in English instead of the Hinid script. Is there a setting that I can change that will enable me to get the output in Hindi itself? 

```
mel = whisper.log_mel_spectrogram(audio).to(model.device)
options = whisper.DecodingOptions(language = 'hi')
result = whisper.decode(model, mel, options)
print(result.text)
```

`Karnataka mein Hindutva etna bhaad ke urai. یہ hai kontroversial Idga Mehdan Karnataka ke hubli mein. Kontroversial islea ki ki 1994 mein, BJP ke kai popular leaders, iss Mehdan ki `

**Comment by jianfch** [comment]
you can try
```python
options = whisper.DecodingOptions(language = 'hi', task='transcribe')
```

**Reply by maugas** [reply]
Did this work?
options = whisper.DecodingOptions(language = 'hi', task='transcribe')

**Reply by jongwook** [reply]
`task='transcribe'` is the default, so it shouldn't make any difference

**Comment by jongwook** [comment, marked as answer]
It could be a phenomenon similar to #204 and #194; could you try giving a prompt that might have come before the audio?

In general, it's an open research question to reliably control the model to prefer a certain format.

**Reply by maugas** [reply]
I have similar issue #234 , please let me know if there is solution available

**Comment by Sarathharidas** [comment]
Thanks! This worked:

!whisper ""auido_path" --language Hindi --task transcribe

**Reply by jianfch** [reply]
so it seems like beam search can be a solution for this phenomenon besides prompts

**Comment by HardikJain02** [comment]
Hi @Sarathharidas, I want transcription in English Script instead of Hindi for Hindi audio. (Just opposite of you) 

I am using [yt_whisper](https://github.com/m1guelpf/yt-whisper) so that I can directly transcribe the video with vtt file by a youtube link. Please let me know how can I achieve it.

**Reply by sushant-samespace** [reply]
@HardikJain02  I know I am late , but were you able to find anything on this ?

**Reply by itaipee** [reply]
you want Latin transliteration of the Hindi  , like in this  online converter  https://mylanguages.org/hindi_romanization.php
I 'm also looking for something similar ,  i tried several  initial-prompts of Hindi-latin transliteration  , but it di not worked.  
I will try fine-tune in the future  , if i get to it ,

**Reply by Sarathharidas** [reply]
For English script you can do 
!whisper ""auido_path" --language Hindi --task translate @HardikJain02

**Reply by itaipee** [reply]
I don't think that what they meant.   Not Translation , Transliteration, using Latin words for representing the Hindi language 
for example   , मैं क्या मैं आपकी मदद कर सकता हूँ   ,which is translated to 'i can i help you'  according to google translate  ,   is transliterated to "main kyaa main aapki mdd kr sktaa hun"  ,

i found converter online here : https://mylanguages.org/hindi_romanization.php  

There are several application for Latin transliterations , not only for Hindi  , but it is less common so far in speech recognition domain
